Automattic/pocket-casts-ios
# Eoy2024: fix marquee alignment and sizes

| 📘 Part of: #2250  |  
|:---:|

Fixes #2398  


This PR updates the marquee component to better align text and images and have proper spacing between elements.
I also took the opportunity to move the Marquee component to a separate file and add a preview for easier tweak of parameters.

| 1 | 2 | 3 |
| - | - | - |
| ![Simulator Screenshot - iPhone 16 Pro - 2024-11-06 at 10 19 22](https://github.com/user-attachments/assets/872e423e-3943-4091-afb3-46a6f1c60410) | ![Simulator Screenshot - iPhone 16 Pro - 2024-11-06 at 10 17 07](https://github.com/user-attachments/assets/2fccf073-7f33-4067-9722-70fef533c456) | ![Simulator Screenshot - iPhone 16 Pro - 2024-11-06 at 10 16 26](https://github.com/user-attachments/assets/70439d97-42aa-40e5-9cd9-76a82f7c753b) |


## To test

1. Star the app
2. Ensure you have the EOY 2024 FF active
3. Go to Profile tab and tap on the EOY 2024 banner
4. Check the Epilogue and empty Ratings screen where the marquee is used
5. If you have Ratings and want emulate empty ratings I suggest changing the check on `Ratings2024Story.swift` line 23.

## Checklist
